internal/device/dryRun.go

```diff
@@ -1,15 +1,41 @@
 package device
 
-import "context"
+import (
+	"context"
+	"errors"
+
+	"github.com/metal-toolbox/flipflop/internal/model"
+)
+
+type serverState struct {
+	power      string
+	bootDevice string
+}
+
+var (
+	errBmcCantFindServer = errors.New("dryrun BMC couldnt find server to set state")
+	serverStates         map[string]serverState
```

Comment:
you could initialize this here via `serverStates = map[string]serverState{}` if you wanted and save the nil check below.